api/v1/storagecluster_types.go

```diff
@@ -216,6 +217,13 @@ type ManageCephToolbox struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// MgrSettings defines the settings for the Ceph Manager
+type MgrSettings struct {
```

Comment:
```suggestion
type MgrSpec struct {
```

Lets rename this

Reply:
Avoided naming it as MgrSpec as MgrSpec is an actual struct in rook api, And it has its specific field. Naming it MgrSpec may create confusion. See here
https://pkg.go.dev/github.com/rook/rook@v1.10.0/pkg/apis/ceph.rook.io/v1#MgrSpec

Reply:
MgrSpec seems more consistent, other structs defined in these types have the `Spec` suffix even if it is the same as a Rook type. Since it's in a different package, there isn't a conflict.

Reply:
Ack, Done